src/pages/_error.jsx

```diff
@@ -68,8 +75,33 @@ export async function getServerSideProps({ req, res, err }) {
     res.setHeader('Cache-Control', 's-maxage=86400')
   }
 
+  /**
+   * Resolve the next/dynamic component so we can access the layout component itself,
+   * and subsequently the static .rivetParams via fetchLayoutProps
+   */
+  let layoutProps = {}
+
+  try {
+    if (proxiedProductSlug) {
+      const layout = resolve(
+        await proxiedLayouts[proxiedProductSlug].render.preload()
```

Comment:
`next/dynamic` hangs a `.preload()` method off of its dynamic component wrapper so that they can prefetch the JS files programmatically. Can be seen here: https://github.com/vercel/next.js/blob/352e60deddab9667d045b4eca2a77ef25a8adbaa/packages/next/shared/lib/loadable.js#L156

I think we need to access `.render` here as well because it's wrapped in a `React.forwardRef` call.